lib/rules/sort-invocations.js

```diff
@@ -226,18 +226,42 @@ export default class SortInvocations extends Rule {
           return;
         }
 
-        node = b.block(path, params, sortHash(hash), b.blockItself());
+        if (this.mode === 'fix') {
+          node = b.block(path, params, sortHash(hash), b.blockItself());
+        } else {
+          this.log({
+            isFixable: true,
+            message: 'Hash keys must be sorted',
+            node,
+          });
+        }
       },
 
       ElementNode(node) {
         const { attributes, modifiers } = node;
 
         if (!canSkipSortAttributes(attributes)) {
-          node.attributes = sortAttributes(attributes);
+          if (this.mode === 'fix') {
+            node.attributes = sortAttributes(attributes);
+          } else {
+            this.log({
+              isFixable: true,
+              message: 'Arguments/attributes must be sorted',
+              node,
+            });
+          }
         }
 
         if (!canSkipSortModifiers(modifiers)) {
-          node.modifiers = sortModifiers(modifiers);
+          if (this.mode === 'fix') {
+            node.modifiers = sortModifiers(modifiers);
+          } else {
+            this.log({
+              isFixable: true,
+              message: 'Modifiers must be sorted',
```

Comment:
let's print correct order of modifiers (by name) to improve DX, kinda - `sortModifiers([...modifiers]).map((m => m.path.original)).join(', ');

Reply:
Done.